### DISCUSSION TITLE
Implementing multiple live streams at the same time

### DISCUSSION THREAD

**Discussion by JankiMandviya** [aiortc/aiortc#1222] [Q&A]
I need help with modifying this code to implement 4 live streams with different contents (video1 , video2 , video3 ,video4).
https://github.com/aiortc/aiortc/tree/main/examples/webcam
I am using above source for my purpose. I am trying to do multiple streams at the same time with different contents on different ports.
stream 1 --> port 8080
stream 2 --> port 8081
stream 3 --> port 8082
stream 4 --> port 8083
How can i achieve it.
Also, do i need to make any changes in its respective client.js and index.html?

**Comment by shenwpo** [comment]
You only need to add multiple traces on the js and python.
